src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorProjectChangePublisher.cs

```diff
@@ -18,11 +19,11 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor
     /// <summary>
     /// Publishes project.razor.json files.
     /// </summary>
-    [Shared]
+    [System.Composition.Shared]
     [Export(typeof(ProjectSnapshotChangeTrigger))]
     internal class DefaultRazorProjectChangePublisher : ProjectSnapshotChangeTrigger
     {
-        internal readonly Dictionary<string, Task> _deferredPublishTasks;
+        internal readonly Dictionary<string, System.Threading.Tasks.Task> _deferredPublishTasks;
```

Comment:
Unfortunately there's also a Task on `Microsoft.VisualStudio.Shell` so we're doing this stuff.

Reply:
```csharp
using Task = System.Threading.Tasks.Task;
```
should do it I believe